core/geometry/src/serialization/GeometrySamples.ts

```diff
@@ -2837,4 +2837,32 @@ export class Sample {
     }
     return builder.claimPolyface(true);
   }
+  /**
+   *  alternately move in directions in the steps array, creating numStroke total strokes.
+   * * if start is a single point, create a new point array with start as its first entry.
+   * * if start is an array, add to it.
+   * * if start is an empty array, create a new one and start at 000
+   * @param start
```

Comment:
Would you mind adding some description like "start of the zig zag" for all aprameters? Otherwise, dont mention them at all because it's clear from API itself